packages/workbox-broadcast-cache-update/MESSAGE_TYPES.mjs

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Instead of `MESSAGE_TYPES.mjs`, could you name this file `constants.mjs` and use a named `MESSAGE_TYPES` export?

Or just name this file `messageTypes.mjs` and keep the default export?